composer.json

```diff
@@ -34,7 +34,7 @@
         }
     ],
     "require": {
-        "php": "~5.4|~7.0",
+        "php": ">=7.0",
```

Comment:
i'm not a fan of unbound version constraints as it can lead to bc breaks with newer language versions. maybe we just stick to the constraint that is already defined?

Reply:
Sorry, this was actually meant to be the caret "^". However I can keep the tilde "~" if you want?